src/main/java/org/myrobotlab/service/interfaces/SpeechSynthesisControl.java

```diff
@@ -0,0 +1,18 @@
+package org.myrobotlab.service.interfaces;
```

Comment:
this has nothing to do with the service type / metadata change. please revert from this pr.